api/serverApiFunctions.test.tsx

```diff
@@ -0,0 +1,50 @@
+'use server';
+import { deleteUser } from './serverApiFunctions';
```

Comment:
Q: in a testing environment, `'use server'` is still needed? Could you please explain to me why this is needed in this test case? :)

Reply:
A: Because it still uses the server SDK and needs the user server directive in this file.